sealtk/core/AbstractProxyModel.cpp

```diff
@@ -58,8 +68,7 @@ bool AbstractProxyModel::lessThan(
     // String comparisons
     case core::NameRole:
     case core::UniqueIdentityRole:
-      return QString::localeAwareCompare(left.toString(),
-                                         right.toString());
+      return localeAwareLessThan(left.toString(), right.toString());
```

Comment:
Oh man... yep, that'll do it.

Any reason why you created a local function and didn't just put `< 0` right here?

Reply:
Mainly because the line was getting long and ugly :wink:.